op-challenger/flags/txmgr.go

```diff
@@ -0,0 +1,120 @@
+package flags
+
+import (
```

Comment:
This file shouldn't be needed any more. Can directly use `txmgr.CLIFlags(envVarPrefix)`. This looks like it may have branched out of develop a while ago b/c the txmgr flags have been cleaned up from this.

Reply:
Ah yep that's what happened, thanks!